packages/app/components/Today/SandboxApplication.tsx

```diff
@@ -0,0 +1,51 @@
+import { Modal, Button } from "antd";
+import { ReactElement } from "react";
+import { useLocalStorage } from "../../hooks/useLocalStorage";
+
+export default function ApplyToSandbox(): ReactElement {
+  const [firstTime, setFirstTime] = useLocalStorage("seenApplyModal", true);
+
+  return (
+    <Modal
+      visible={firstTime}
```

Comment:
I wonder if we have some constants APPS_OPEN and APPS_CLOSE and we render if we are between those dates and it's the first time

(So people won't see it in summer 2 once apps are over)